cicd/3-app/load-test.template.yml

```diff
@@ -0,0 +1,87 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: The template used to create the required services for load testing
+Parameters:
+  LoadTestImage:
+    Type: String
+    Description: Load testing docker image. This should be the full name of the image.
+Resources:
+  ECSCluster:
+    Type: 'AWS::ECS::Cluster'
+    Properties:
+      CapacityProviders:
+        - FARGATE
+        - FARGATE_SPOT
```

Comment:
Are you looking to use `FARGATE SPOT`?

Reply:
I think this was a copy/paste error, I will remove it.